docs/mast/mast.rst

```diff
@@ -1353,6 +1417,20 @@ The basic MAST query function returns query results as an `~astropy.table.Table`
    Length = 77 rows
 
 
+The 'params' parameter for the `~astroquery.mast.MastClass.service_request` method requires a key-value pair
```

Comment:
Update this example to reference this page: https://mast.stsci.edu/api/v0/_services.html 

Mention that other services (particularly JWST services) require 2 principal keywords before the actual params (and those are `columns` and `filters` - see link above for more context.